dataeng/resources/dbt-run.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+set -ex
+
+# Setup to run dbt commands
+cd $WORKSPACE/warehouse-transforms
+
+# Using --first-parent flag helps to avoids any intermediate commits that developers sometimes leave and merge without squashing. It helps us ignoring the noise in commit history.
+HEAD_COMMIT=$(git log --first-parent origin/master --format='%H' --max-count=1)
+
+if git log --format=%B -n 1 $HEAD_COMMIT | grep 'Schema Builder automated dbt update at' -q; then isSchemaBuilderPR="true"; else isSchemaBuilderPR="false"; fi
```

Comment:
other nit: I think we should standardize on one format for variables. `IS_SCHEMA_BUILDER_PR`, for example.

Reply:
yes we should, done